.github/workflows/reusable-tox.yml

```
@@ -272,19 +274,25 @@ jobs:
         path: dist/
 
     - name: >-
-        Pre-populate tox envs: `${{ env.TOXENV }}`
-      run: >-
-        python -Im
-        tox
-        --parallel auto
-        --parallel-live
-        --skip-missing-interpreters false
-        ${{
-          inputs.built-wheel-names != ''
-          && format('--installpkg dist/{0}', inputs.built-wheel-names)
-          || ''
-        }}
-        --notest
+        Pre-populate tox envs: ${{ env.TOXENV }}
+      shell: bash
+      env:
+        INPUT_BUILT_WHEEL_NAMES: ${{ inputs.built-wheel-names }}
+      run: |-
+        tox_common_args=(
+          --parallel auto
+          --parallel-live
+          --skip-missing-interpreters false
+        )
+        if [ -n "$INPUT_BUILT_WHEEL_NAMES" ]; then
```

With `shell: bash` this probably can use Bashism:
```suggestion
        if [[ $INPUT_BUILT_WHEEL_NAMES ]]; then
```
Else there should be not reason for Bash specifically (POSIX compatible `/bin/sh` should be okay).